docs/apis/migration-perm-guidance.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Migration permission guidance
+ms.author: jhendr
+author: JoanneHendrickson
+manager: pamgreen
+audience: ITPro
+f1.keywords:
+- NOCSH
+ms.topic: conceptual
+ms.service: sharepoint-online
+localization_priority: Priority
+ms.collection: 
+- SPMigration
+- M365-collaboration
+search.appverid: MET150
+description: "Migration permission guidance"
+---
+# Migration permission guidance
+
+## Understanding security scope, role assignments and item limits during migration
```

Comment:
Starting a doc with nothing before the H2 isn't recommended (believe it triggers build issues). Suggest adding text before this heading summarizing what the doc is about.